### ISSUE TITLE
The each() function is deprecated.

### ISSUE THREAD

**Issue by flyingcoder** [BingAds/BingAds-PHP-SDK#55]
vendor/microsoft/bingads/src/Auth/HttpService.php(18): each(Array) on PHP 7.2

Do you have any idea how to fix this for a moment?

**Comment by eric-urban** [comment]
@flyingcoder thanks for raising this issue! The following replacement should work:

```php
foreach ($postData as $key => $val) {
    if(strlen($query) > 0)
    {
        $query = $query . '&';
    }

    $query = $query . $key . '=' . rawurlencode($val);
}
```

Will verify and plan to include it in the next release.

**Comment by eric-urban** [comment]
@flyingcoder this is fixed in v11.12.1
